core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -188,6 +188,12 @@ class DAGScheduler(
   private val disallowStageRetryForTest = sc.getConf.getBoolean("spark.test.noStageRetry", false)
 
   /**
+   * If enabled, fetch failure will cause all the output on that host to be unregistered.
```

Comment:
Rephrase this by
```
Whether to unregister all the outputs on the host in condition that we receive a FetchFailure, this is set default to false, which means, we only unregister the outputs related to the exact executor(instead of the host) on a FetchFailure.
```